### PR TITLE
fix(integration-test): adapting to Camel3 (version and http4)

### PR DESCRIPTION
**Pull request by Delawen**
Changing versions for variables so we always test the version we are working on. Removing all HTTP4 reference (no longer in Camel 3.x)

### CODE REVIEW

File: app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
           "authenticationParameterName": "keyName",
           "authenticationParameterPlacement": "header",
           "authenticationType": "apiKey:api_key",
-          "componentName": "connector-rest-swagger-http4",
+          "componentName": "connector-rest-swagger-ttp",
```
Comment (oscerd) [inline app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json]: I think it is connector-rest-swagger-http

File: app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json [hunk 2]
```diff
@@ -343,7 +343,7 @@
         "authenticationParameterName": "keyName",
         "authenticationParameterPlacement": "header",
         "authenticationType": "apiKey:api_key",
-        "componentName": "connector-rest-swagger-http4",
+        "componentName": "connector-rest-swagger-ttp",
```
Comment (oscerd) [inline app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json]: I think it is connector-rest-swagger-http

File: app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json [hunk 3]
```diff
@@ -826,7 +826,7 @@
                     "authenticationParameterName": "keyName",
                     "authenticationParameterPlacement": "header",
                     "authenticationType": "apiKey:api_key",
-                    "componentName": "connector-rest-swagger-http4",
+                    "componentName": "connector-rest-swagger-ttp",
```
Comment (oscerd) [inline app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json]: ditto

File: app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json (hunk 1 above)
Comment (Delawen) [reply app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json]: Yeap, fixed.

**Pull request merged**